pkg/probe/exec/exec.go

```diff
@@ -66,7 +66,13 @@ func (pr execProber) Probe(e exec.Cmd) (probe.Result, string, error) {
 			if exit.ExitStatus() == 0 {
 				return probe.Success, string(data), nil
 			}
-			return probe.Failure, string(data), nil
+			probeFailedMsg := string(data)
+			if len(probeFailedMsg) > 0 && !strings.HasSuffix(strings.Trim(probeFailedMsg, " "), "\n") {
+				probeFailedMsg += "\n"
```

Comment:
@NoicFank this is a bit unclear to me. We're trimming message in the check, but then we add "\n" to the original(not trimmed) message. So, for the string "Error: some error \n   " we'll add extra "\n" to the end.
Can you explain why is this needed?

Reply:
nvm, I understand it now. Makes sense to me.

Reply:
> nvm, I understand it now. Makes sense to me.

thanks for review 😊